app/subscriber/src/features/filter-media/FilterMedia.tsx

```diff
@@ -0,0 +1,120 @@
+import { MsearchMultisearchBody } from '@elastic/elasticsearch/lib/api/types';
+import { DateFilter } from 'components/date-filter';
+import { determineColumns } from 'features/home/constants';
+import { createFilterSettings } from 'features/press-gallery/utils';
+import moment from 'moment';
+import React from 'react';
+import { FiRefreshCcw } from 'react-icons/fi';
+import { useNavigate } from 'react-router-dom';
+import { useContent, useLookup, useLookupOptions } from 'store/hooks';
+import {
+  FieldSize,
+  FlexboxTable,
+  generateQuery,
+  IContentModel,
+  IFilterSettingsModel,
+  IOptionItem,
+  Row,
+  Select,
+} from 'tno-core';
+
+import * as styled from './styled';
+
+export const FilterMedia: React.FC = () => {
+  const navigate = useNavigate();
+  const [{ filterAdvanced }, { findContentWithElasticsearch }] = useContent();
+  const [{ productOptions, sourceOptions }] = useLookupOptions();
+  const [{ sources, products }] = useLookup();
+  const [results, setResults] = React.useState<IContentModel[]>([]);
+
+  const [productValue, setProductValue] = React.useState<IOptionItem | null>();
+  const [sourceValue, setSourceValue] = React.useState<IOptionItem | null>();
+
+  const [settings, setSettings] = React.useState<IFilterSettingsModel>(
+    createFilterSettings(
+      filterAdvanced.startDate ?? moment().startOf('day').toISOString(),
+      filterAdvanced.endDate ?? moment().endOf('day').toISOString(),
+    ),
+  );
+
+  const fetchResults = React.useCallback(
+    async (filter: MsearchMultisearchBody) => {
+      try {
+        const res: any = await findContentWithElasticsearch(filter, false);
+        setResults(res.hits.hits.map((h: { _source: IContentModel }) => h._source));
+      } catch {}
+    },
+    [findContentWithElasticsearch],
+  );
+
+  React.useEffect(() => {
+    setSettings(
```

Comment:
Curious if you see the double event when you load the page with this?  You don't need to change anything, but I suspect there is one.